docs/source/install/upgrades.rst

```diff
@@ -43,16 +43,22 @@ Upgrade Procedure
 
      sudo yum update $PKG_NAME
 
-3. Run the migration script (if any). See section below for |st2| 
-   version-specific migration scripts.
-
-4. Upgrade Mistral database.
+3. Upgrade Mistral database.
 
 .. sourcecode:: bash
 
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
```

Comment:
I guess these 2 DB upgrade commands should be run on every new st2mistral version (eg the script won't fail if DB is already latest and it will upgrade if DB upgrade is really needed).

What if we include this logic in `st2mistral` packaging?

```sh
# best effort
# don't hang entire package upgrade with error if some of the commands failed (ex: postgresql is not yet available)
on every `st2mistral` package upgrade:
  stop mistral services
  mistral-db-manage upgrade head
  mistral-db-manage populate
  start mistral services
```

Any corner cases? WDYT?